packages/fxa-auth-server/lib/routes/utils/otp.js

```diff
@@ -16,21 +16,18 @@ module.exports = (log, config, db) => {
      * @param account
      * @returns boolean
      */
-    hasTotpToken(account) {
+    async hasTotpToken(account) {
       const { uid } = account;
-      return db.totpToken(uid).then(
-        result => {
-          if (result && result.verified && result.enabled) {
-            return true;
```

Comment:
Just noticed that the way this is written means it could return undefined if the result here doesn't satisfy this condition.